pelican/tools/pelican_quickstart.py

```diff
@@ -4,10 +4,14 @@
 import locale
 import os
 from typing import Mapping
+from zoneinfo import available_timezones
```

Comment:
This is unnecessary *and* unused.

Reply:
Ah yeah got to love automatic imports haha I'll remove it thanks